vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/Dashboard.java

```diff
@@ -143,6 +143,18 @@ public void removeAll() {
         updateClient();
     }
 
+    /**
+     * Sets the maximum column count of the dashboard.
+     *
+     * @param maxCount
+     *            the new maximum column count. Pass in {@code null} to set the
+     *            maximum column count back to the default value.
+     */
+    public void setMaximumColumnCount(Integer maxCount) {
+        getStyle().set("--vaadin-dashboard-col-max-count",
+                maxCount == null ? null : String.valueOf(maxCount));
+    }
+
```

Comment:
Shouldn't there also be a getter?

Reply:
I wasn't sure about that since there are examples like [setOverlayWidth in ComboBox](https://github.com/vaadin/flow-components/blob/main/vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java#L332-L341) where there is no getter. I can add it if we decide it should be there.

Reply:
Added the getter and unit tests for it.